docs/Analyzer Configuration.md

```diff
@@ -512,3 +512,25 @@ Option Values: One or more fields of enum [Microsoft.CodeAnalysis.SymbolKind](ht
 Default Value: Namespace, NamedType, Method, Property, Event, Parameter
 
 Example: `dotnet_code_quality.CA1716.analyzed_symbol_kinds = Namespace, Property`
+
+### Additional use results methods
+Option Name: `additional_use_results_methods`
+
+Configurable Rules: [CA1806](https://docs.microsoft.com/en-us/visualstudio/code-quality/CA1806)
+
+Option Values: Names of additional methods (separated by '|') for CA1806.
```

Comment:
Do we want to also allow types? Same question for namespaces? IMO types could make sense but I fail to see any point for namespaces.